api/src/core/BackgroundTasks.php

```diff
@@ -487,9 +487,13 @@ private function updatePlugin($plugin, $index = null, $length = null, $subtasks)
    }
 
    private function alertWatchers($plugin) {
-      $client_url = Tool::getConfig()['client_url'];
+      $mime_grammar = new \Swift_Mime_Grammar();
+
       foreach ($plugin->watchers()->get() as $watch) {
          $user = $watch->user;
+         if (!preg_match('/^'.$mime_grammar->getDefinition('addr-spec').'$/D', $user->email)) {
```

Comment:
Uses same validation logic as in https://github.com/swiftmailer/swiftmailer/blob/5.x/lib/classes/Swift/Mime/Headers/MailboxHeader.php#L341